kubernetes/Razee/razeedeploy.yaml

```
@@ -0,0 +1,57 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: razee
```

would you be against putting this in the razeedeploy namespace. that is the default and where the razeedash-api is going to put it when it installs it in a later step.
```suggestion
  name: razeedeploy
```